src/content/ja/billions/_index.yaml

```diff
@@ -1,150 +1,142 @@
-# ATTENTION - changes made to this file not in GitHub will be over written!
-
-project_path: /web/_project.yaml
-book_path: /web/_book.yaml
+---
```

Comment:
This line needs to be removed

Reply:
This might be an issue on GitLocalize.
@ilyaspiridonov can you take a look?